GliaWidgets/Sources/ViewModel/Chat/ChatViewModel+ChoiceCards.swift

```diff
@@ -0,0 +1,63 @@
+import Foundation
+import SalemoveSDK
+
+extension ChatViewModel {
+    func sendChoiceCardResponse(_ option: ChatChoiceCardOption, to messageId: String) {
+
+        guard let option = option.asSingleChoiceOption() else { return }
+        environment.sendSelectedOptionValue(option) { [weak self] result in
+            guard let self = self else { return }
+
+            switch result {
+            case .success(let message):
+                guard
+                    let selection = message.attachment?.selectedOption
+                else { return }
+
+                self.respond(to: messageId, with: selection)
+
+            case .failure:
+                self.showAlert(
+                    with: self.alertConfiguration.unexpectedError,
+                    dismissed: nil
+                )
+            }
+        }
+    }
+
+    private func respond(to choiceCardId: String, with selection: String?) {
+        guard let index = messagesSection.items
```

Comment:
- since there could be a lot of items, perhaps we can optimize a little bit this enumeration and make it lazy? would it make sense?
```swift
guard let index = messagesSection.items.lazy
```